src/components/Button/Button.js

```diff
@@ -1,13 +1,29 @@
-import './Button.scss';
+import "./Button.scss";
 
-import React from 'react';
-import { confirmAlert } from 'react-confirm-alert';
-import 'react-confirm-alert/src/react-confirm-alert.css';
+import React from "react";
+import { confirmAlert } from "react-confirm-alert";
+import "react-confirm-alert/src/react-confirm-alert.css";
 
 const Button = (props) => {
-  const { className, onClickHandler, ButtonIcon, buttonImage, buttonImageAltText, buttonText, buttonOuter, buttonOuterClassName, disabled, confirmText, title, label } = props;
-  
-  const buttonClass = `btn${className ? ` ${className}` : ""}${buttonText ? "" : " btn--svg-only"}`
+  const {
+    className,
+    onClickHandler,
+    ButtonIcon,
+    buttonImage,
+    buttonImageAltText,
+    buttonText,
+    buttonOuter,
+    buttonOuterClassName,
+    disabled,
+    confirmText,
+    title,
+    label,
+    buttonIconLeft = true,
```

Comment:
is there a `buttonIconRight`? only asking as the naming is a little confusing if what it's there for is to toggle visibility of an icon, perhaps `buttonIconVisible` could be a little clearer, but I realise I could be misunderstanding the full intention :) 

A further thought since it's true by default, is it could also be clearer to invert the logic, so it's only set if the icon should be hidden, so `buttonIconHidden`? :D

Reply:
I think it's the implicitness of the behaviour that bugs me, maybe if it was called `moveIconLeft` or something? I was expecting if it was false there'd be no icon. Another approach could be to pass a value so you could use it like `buttonIcon = 'left'` which would also allow it to potentially be false and show no icon, but now I get what you're trying to achieve, I'll stop meddling, haha.

Reply:
I'll have a think and give it a rejig, you're right though the implicit behaviour for moving it to the right isn't great

Reply:
done 👍